platform/ui/src/components/Icon/icon.mdx

```diff
@@ -0,0 +1,43 @@
+---
+name: Icon
+menu: Components
+route: components/icon
+---
+
+import { Playground, Props } from 'docz';
+import Icon from './Icon.js';
+import { ICONS } from './getIcon.js';
+
+# Icon
+
+## Basic usage
+
+<Playground>
+  <Icon name="active" className="text-custom-aquaBright m-4" />
+</Playground>
+
+## All Icons
+
+<div className="flex flex-wrap items-center content-center bg-black">
+  {Object.keys(ICONS)
+    .sort()
+    .map((name, i) => {
+      return (
+        <React.Fragment key={i}>
+          <div className="flex flex-col m-1 p-4 items-center">
```

Comment:
Flexbox appears off. I'm seeing some odd alignment

Reply:
yeah was because of some big names, just fixed the size and its more fluid now, let me know if you still think we can change.